src/rpc/util.cpp

```diff
@@ -749,3 +750,69 @@ UniValue GetServicesNames(ServiceFlags services)
 
     return servicesNames;
 }
+
+UniValue CanWriteFile(const std::string& filepath)
```

Comment:
maybe rename to `EnsureFileWritable(const std::string& filepath)`, then move the `throw`s inside, returning void. This makes the call sites much cleaner, and there is precedent in other RPC utility functions such as `EnsureMemPool` `EnsureWallet` etc

Reply:
Thanks. Will adapt the code.